lib/src/misc.dart

```diff
@@ -110,3 +110,31 @@ class _PreviousHookState<T> extends HookState<T, _PreviousHook<T>> {
   @override
   T build(BuildContext context) => previous;
 }
+
+/// Runs the callback on every hot reload
+/// similar to reassemble in the Stateful widgets
+/// See also:
+///
+///  * [State.reassemble]
+void useReassemble(VoidCallback callback) =>
+    Hook.use(_ReassembleHook(callback));
```

Comment:
It's probably a good idea to wrap that `Hook.use` into an `assert`. This way, the hook disappears in release mode

Reply:
by doing that i have to change `Hook<void>` to `Hook<bool>` ( or other types ) is it ok or is there a better way?